package.json

```diff
@@ -34,6 +34,7 @@
     "typescript": "~4.5.2"
   },
   "dependencies": {
-    "esbuild": "^0.14.0"
+    "esbuild": "^0.14.0",
+    "reflect-metadata": "^0.1.13"
```

Comment:
I noticed the packages/-lock.json files changed. I was wondering if the `reflect-metadata` not included with previous merges?